### ISSUE TITLE
Test CanCreateASignedCommit fails in master branch

### ISSUE THREAD

**Issue by AArnott** [libgit2/libgit2sharp#1313]
This test fails in the master branch (as of 1fb7fc5b1d26c893e085c2782d5ee483d56c5a18)

```
Test Name:	LibGit2Sharp.Tests.CommitFixture.CanCreateASignedCommit
Test FullName:	LibGit2Sharp.Tests.CommitFixture.CanCreateASignedCommit
Test Source:	C:\Users\andarno\git\libgit2sharp\LibGit2Sharp.Tests\CommitFixture.cs : line 1157
Test Outcome:	Failed
Test Duration:	0:00:00.017

Result StackTrace:	
at LibGit2Sharp.Core.Ensure.HandleError(Int32 result) in C:\Users\andarno\git\libgit2sharp\LibGit2Sharp\Core\Ensure.cs:line 154
   at LibGit2Sharp.Core.Ensure.ZeroResult(Int32 result) in C:\Users\andarno\git\libgit2sharp\LibGit2Sharp\Core\Ensure.cs:line 172
   at LibGit2Sharp.Core.Proxy.git_commit_create_with_signature(RepositoryHandle repo, String commitContent, String signature, String field) in C:\Users\andarno\git\libgit2sharp\LibGit2Sharp\Core\Proxy.cs:line 441
   at LibGit2Sharp.ObjectDatabase.CreateCommitWithSignature(String commitContent, String signature) in C:\Users\andarno\git\libgit2sharp\LibGit2Sharp\ObjectDatabase.cs:line 448
   at LibGit2Sharp.Tests.CommitFixture.CanCreateASignedCommit() in C:\Users\andarno\git\libgit2sharp\LibGit2Sharp.Tests\CommitFixture.cs:line 1163
Result Message:	LibGit2Sharp.LibGit2SharpException : malformed commit contents
```

**Comment by ethomson** [comment]
That's very surprising, and all our CI is passing here.  I _would_ expect this if you were using a slightly older libgit2.  Just to be clear:  is this with a stock setup?  Can you tell me which libgit2.NativeBinaries nuget package you're using?

**Comment by AArnott** [comment]
I presume it's using the one that packages.config prescribes, as nuget restore downloaded it:

```xml
<package id="LibGit2Sharp.NativeBinaries" version="1.0.137" targetFramework="net4" allowedVersions="[1.0.137]" />
```

**Comment by ethomson** [comment]
Aha.  Looks to be EOL translation issues.

**Comment by ethomson** [comment]
Fixed via #1315
